Pipeline/SimpleBuildReportDeploy/README.md

```diff
@@ -0,0 +1,293 @@
+# Simple Build Report Deploy (Deploy the package to target libraries)
+## Description
+This "SimpleBuildReportDeploy.groovy" sample script is capable of untaring the tar package produced by [PackageBuildOutputs](../PackageBuildOutputs/) and deploy the tar package contents to the target libraries. 
+
+Potential scenarios include:
+* Deploying the tar package to the target environment libraries.
+* If the user wants to do shift-left testing using IBM Z Virtual Test Platform, and subsystem updates are not required.
+
+As of now, this script does not perform any activation activities like a CICS NEWCOPY or a DB2 BIND.  Currently this script does not support rollback process also.
+
+### Prerequisites
+
+* All the target environment libraries are existing.
+* The deployment package is available in USS. (For example, it could be a downloaded copy from the binary artifact repository.)
+* The user invoking the script has proper permissions to write to the target datasets.
+
+### Installation
+* Copy the below sample scripts into USS files under the dbb-zappbuild folder.
+* Update the permission bits to allow for execute. Ex: chmod 755 *
+
+### Customization/Configuration
+* Review the notes sections in each sample script prologue for more information on customization.
+
+### Sample invocation
```

Comment:
Please generalise your userid ( like using IBMUSER ) including the zAppBuild_Hogan path in the sample invocation outputs.

Reply:
This is done and created a new pull request #148 .

Reply:
Please ignore #148 and consider #149 .